week-2-wide-perspective/w02e05-simple-server-side/src/modules/products/products.controller.ts

```diff
@@ -0,0 +1,35 @@
+import { Request, Response } from "express";
+import { prisma } from "../../../prisma/db";
+
+export const getProducts = async (req: Request, res: Response) => {
+  try {
```

Comment:
ten `try` `catch` też nie będzie potrzebny jeśli zrobisz "globalErrorHandler" + do aplikacji dodasz to:

https://www.npmjs.com/package/express-async-errors

jako `import` w `app.ts` 
- to jest konieczne żeby błędy `async` były łapane przez Default Error Handler